app/javascript/.storybook/preview.js

```diff
@@ -66,9 +65,16 @@ const themeSwitcherDecorator = (storyFn) => {
 };
 
 addDecorator(themeSwitcherDecorator);
-addDecorator(withA11y);
 
 addParameters({
+  docs: {
```

Comment:
I believe this is related to switching to the addon-docs, so let's pull this out for now and as mentioned above, let's do this in another PR.